src/api/authApi.js

```diff
@@ -0,0 +1,65 @@
+
+import router from '@/router'
+import axios from 'axios'
+
+export const URL_KEY = 'AIzaSyDLzMdBM3yeRBVTM3z6fbSh3p8TYOq-0jQ'
+
+export const axiosAuth = axios.create({
+  baseURL: 'https://identitytoolkit.googleapis.com/v1'
+})
+
+axiosAuth.interceptors.request.use(config => {
+  return config
+})
+
+axiosAuth.interceptors.response.use(response => {
+  return response
+}, function (error) {
+  if (error.response.status === 400 && error.response.data.error.message === 'EMAIL_EXISTS') {
```

Comment:
Request / Response interceptors are being used for general errors only (like you did in dataApi.js). All other errors like:
'EMAIL_EXISTS' you need to handle inside component only. Inside /login or /signup page.